### PR TITLE
[TC-7341] Remove media types docs and add document storage security docs

### PR DESCRIPTION
**Pull request by roy-tc**
## Description

https://tradecloud.atlassian.net/browse/TC-7341

### Scope
This PR updates the security documentation according to the latest implementation in the object-storage. File extensions are no longer checked, instead docs are added about how documents are protected.

Can be reviewed here: https://app.gitbook.com/@tradecloud/s/api/v/TC-7341-remove-filevalidator/security/documents

### Submitter pre-flight check

- [x] Review your documentation
- [x] Add labels `needs reviewing`
- [x] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

File: security/documents.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Document Storage
+
+Documents that are uploaded to our Object Storage ([buyer](../buyer/issue/attach-document.md#step-1-upload-a-document-to-the-tradecloud-object-storage) / [supplier](../supplier/send-order-response/attach-document.md#step-1-upload-a-document-to-the-tradecloud-object-storage)) are securely stored in our cloud.
+
+We protect your documents and the users that receive your documents as follows:
+
+* Documents can only be uploaded or downloaded through an [encrypted connection](encryption.md)
+* Once uploaded, documents are immediately scanned for viruses, worms and trojans. The scanner's database is updated every 3 hours.
+* Only authenticated Tradecloud users can request a download link of a document.
+* If a Tradecloud user requests a download link, we first verify that the user is [authorized](authorization.md) to access this document.
```
Comment (marcmatt) [inline security/documents.md:10]: Make it 1 sentence:
Only authenticated and authorized Tradecloud users can request a download link of a document.
(link to both authentication and authorization pages)

File: security/documents.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Document Storage
+
+Documents that are uploaded to our Object Storage ([buyer](../buyer/issue/attach-document.md#step-1-upload-a-document-to-the-tradecloud-object-storage) / [supplier](../supplier/send-order-response/attach-document.md#step-1-upload-a-document-to-the-tradecloud-object-storage)) are securely stored in our cloud.
+
+We protect your documents and the users that receive your documents as follows:
+
+* Documents can only be uploaded or downloaded through an [encrypted connection](encryption.md)
+* Once uploaded, documents are immediately scanned for viruses, worms and trojans. The scanner's database is updated every 3 hours.
```
Comment (marcmatt) [inline security/documents.md:8]: Add: 
- Documents are stored encrypted
- Documents are stored spanning multiple EU data centers.

File: security/encryption.md [hunk 3]
```diff
@@ -16,5 +16,5 @@ Check the quality of the Tradecloud certificate using [SSL Labs](https://www.ssl
 
 ## Encrypted and hashed tokens
 
-The Tradecloud API uses hashed [JSON Web Tokens](https://jwt.io/) therefor the token cannot be changed.
+The Tradecloud API uses hashed [JSON Web Tokens](https://jwt.io/) therefore the token can only be compromised.
```
Comment (marcmatt) [inline security/encryption.md:19]: What do you mean with "can only be compromised." (does not sound good)

**Review by marcmatt — CHANGES_REQUESTED** [review]
Good idea to add a document storage page.

File: security/encryption.md (hunk 3 above)
Comment (roy-tc) [reply security/encryption.md]: I rephrased this sentence too many times; and ended up with a mess. Should be better now.

File: security/documents.md (hunk 2 above)
Comment (roy-tc) [reply security/documents.md]: Updated

File: security/documents.md (hunk 1 above)
Comment (roy-tc) [reply security/documents.md]: Good idea, updated.

**Comment by roy-tc** [comment]
Thanks for the feedback @marcmatt , can you re-review?

**Review by marcmatt — APPROVED** [review]
Looks great!

**Review by olegtradecloud — APPROVED** [review]

**Pull request merged**